src/Structure/Content/Menu.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Abstract class for a custom menu.
+ *
+ * @author  Chrispian Burks <chrispian.burks@webdevstudios.com>
+ * @package WebDevStudios\OopsWP\Structure\Content
```

Comment:
Let's move the `Menu` and `MenuInterface` files into `Structure\Menu`, then update all of these references. They're not really "content" the same way a `PostType` or `Taxonomy` is.

Reply:
Working on this last bit now. Thanks for the great suggestions on everything here. Super helpful to see the decision process that goes into something like this. I've also altered the style of my code a little bit more to match up how things were done by you and Tom on #25.